jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/exceptions/exceptionhandling/ApiResponseExceptionHandler.java

```diff
@@ -42,9 +42,9 @@ public ResponseEntity<ApiErrorResponse> handleTypeMismatchException(TypeMismatch
 
```

Comment:
Before we merge we will need to discuss versioning. As we have three PR's that need to be merged.